libraries/AP_Mount/AP_Mount.cpp

```diff
@@ -33,6 +32,38 @@ const AP_Param::GroupInfo AP_Mount::var_info[] = {
     AP_GROUPEND
 };
 
+void AP_Mount::Write_Mount(uint8_t instance, enum LogMessages msg, uint64_t timestamp_us)
+{
+    const auto nanf = AP::logger().quiet_nanf();
+    float_t roll = nanf;
+    float_t pitch = nanf;
+    float_t yaw = nanf;
+    float_t target_roll = nanf;
+    float_t target_pitch = nanf;
+    float_t target_yaw = nanf;
+
+    // get mount's actual angles and convert yaw to earth frame
+    get_attitude_euler(instance, roll, pitch, yaw);
+    yaw = wrap_PI(yaw + AP::ahrs().yaw);
+
+    // get mount's target angles and convert yaw to earth frame
+    get_target_attitude_euler(instance, target_roll, target_pitch, target_yaw);
+    target_yaw = wrap_PI(target_yaw + AP::ahrs().yaw);
```

Comment:
This is only logging the earth-frame yaw.  For the actual yaw we need both body-frame and earth-frame logged.  For the target, I think we should also have two fields but for the one that is not actually a target, let's set it to NaN.